arches/app/templates/javascript.htm

```diff
@@ -775,6 +776,13 @@
         count-of-tiles-updated='(count) => {return {% trans "Tiles updated: ${count}" as countOfTilesUpdated %} `{{ countOfTilesUpdated|escapejs }}` }'
         count-of-resources-deleted='(count) => {return {% trans "Resources deleted: ${count}" as countOfResourcesDeleted %} `{{ countOfResourcesDeleted|escapejs }}` }'
         count-of-tiles-deleted='(count) => {return {% trans "Tiles deleted: ${count}" as countOfTilesDeleted %} `{{ countOfTilesDeleted|escapejs }}` }'
+        delete-tile-count='(tile, resource) => {return {% trans "${tile} tiles from ${resource} resources will be deleted" as deleteTileCount %} `{{ deleteTileCount|escapejs }}` }'
+        delete-resource-count='(resource) => {return {% trans "${resource} resources will be deleted" as deleteResourceCount %} `{{ deleteResourceCount|escapejs }}` }'
+        delete-tiles='{% trans "Delete Tiles" as deleteTiles %} "{{ deleteTiles|escapejs }}"'
+        delete-resources='{% trans "Delete Resources" as deleteResources %} "{{ deleteResources|escapejs }}"'
+        to-be-deleted-tiles='{% trans "To Be Deleted Tiles" as toBeDeletedTiles %} "{{ toBeDeletedTiles|escapejs }}"'
```

Comment:
The variable names are okay, but could the text be changed to "Tiles to Be Deleted"?